<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Japanese 練習</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_input">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>200</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>4</number>
    </property>
    <widget class="QLineEdit" name="input_box">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>4</y>
       <width>170</width>
       <height>123</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>48</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="success_rate">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>260</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string>acc</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_reset">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>94</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_output">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1121</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>4</number>
    </property>
    <widget class="QLabel" name="display_label_future_1">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>10</y>
       <width>151</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>60</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>4</number>
     </property>
     <property name="text">
      <string>F1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="display_label_current">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>10</y>
       <width>181</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>60</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>4</number>
     </property>
     <property name="text">
      <string>C</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="display_label_past_1">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>151</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>60</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>4</number>
     </property>
     <property name="text">
      <string>P1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="display_label_past_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>151</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>60</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>4</number>
     </property>
     <property name="text">
      <string>P2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="display_label_past_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>60</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>4</number>
     </property>
     <property name="text">
      <string>P3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="display_label_future_2">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>10</y>
       <width>151</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>60</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>4</number>
     </property>
     <property name="text">
      <string>F2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="display_label_future_3">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>10</y>
       <width>151</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>60</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>4</number>
     </property>
     <property name="text">
      <string>F3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_go_to_selection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>94</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Select kana</string>
    </property>
   </widget>
   <widget class="QLabel" name="kpm">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>190</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string>kpm</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="correct_answer">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>230</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>34</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ca</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_go_to_credits">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>94</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Credits</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
